.github/workflows/Package.yml

```diff
@@ -55,17 +55,38 @@ jobs:
         with:
           python-version: ${{ inputs.python_version }}
 
-      - name: 🔧 Install dependencies for packaging and release
-        run: |
-          python -m pip install -U pip
-          python -m pip install ${{ inputs.requirements }}
+      - name: 🐍 Update pip
+        run: python -m pip install -U pip
 
-      - name: 🔨 Build Python package (source distribution)
+      # build
+
+      - name: 🔧 [build] Install dependencies for packaging and release
+        if: inputs.requirements == ''
+        run: python -m pip install build
```

Comment:
Isn't wheel required too?

Reply:
That is explicitly specified in the `pyprojet.toml`, which is used by `build`. See e.g. https://github.com/edaa-org/pyEDAA.ProjectModel/blob/dev/pyproject.toml#L5.

Reply:
So it works like this:
1. we install `build`
2. it reads the `pyproject.toml`
3. it installs dependencies
4. is does the build itself.